aces/Loris
# [Instrument Builder] updated module help text

New copy for the Instrument Builder module help text

#### Testing instructions

1. Review copy - note that I paired this one down a lot. I felt like the instructions were way too detailed, but I may have taken out too much. Let me know.
2. See screenshot

#### Links to related tickets

* https://github.com/aces/Loris/issues/5576

![Screenshot 2019-11-14 15 19 19](https://user-images.githubusercontent.com/37813606/68892966-26248000-06f2-11ea-9ab8-c08e554dc5e1.png)
